python/conditions.md

```diff
@@ -189,21 +189,21 @@ It’s conventional not to explicitly compare a condition to `True`, because the
 
 ```python
 >>> if ('Pineapple' in pizzas) == True:
-...     print 'Weird.'
+...     (print 'Weird.')
 ...
 >>> if 'Pineapple' in pizzas:
-...     print 'Not weird.'
+...     print ('Not weird.')
```

Comment:
Remove extra space between `print` and `(`

Reply:
Thx for checking ! Done.